### PR TITLE
Changelog for `3.0.0 dev3`

### PR DESCRIPTION
**Pull request by jumaffre**

### CODE REVIEW

**Review by MahatiC — APPROVED** [review]
lgtm!

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - New `snpinfo.sh` script (#4196).
+- New `crypto::hmac` API (#4204).
+
+### Changed
+
+- Application-defined endpoints are now accessible at with both `/app` prefix and un-prefixed, e.g. `GET /app/log/private` and `GET /log/private` (#4147).
```
Comment (eddyashton) [inline CHANGELOG.md:17]:
```suggestion
- Application-defined endpoints are now accessible with both `/app` prefix and un-prefixed, e.g. `GET /app/log/private` and `GET /log/private` (#4147).
```

Comment (MahatiC) [inline CHANGELOG.md:17]:
```suggestion
- Application-defined endpoints are now accessible with both `/app` prefix and un-prefixed, e.g. `GET /app/log/private` and `GET /log/private` (#4147).
```

**Review by MahatiC — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (MahatiC) [reply CHANGELOG.md:17]: Ah I'm a bit late. @eddyashton already did this :D

**Review by eddyashton — APPROVED** [review]

**Pull request merged**